ivy/functional/frontends/torch/tensor_functions.py

```diff
@@ -2,10 +2,6 @@
 import ivy
 
 
```

Comment:
Is this supposed to be deleted?

Reply:
Oops, I'm surprised I didn't notice that. Thanks, I recovered it.